### PR TITLE
chore: Update SDK package version to the latest

### LINKED ISSUE #71: Golang: Update SDK version to the latest and example
With the updated made in https://github.com/momentohq/client-sdk-go/pull/15, need to get the latest SDK version and update the example code.
https://github.com/momentohq/client-sdk-go/issues/16

### PR DESCRIPTION
**Pull request by poppoerika**
Output of running `main.go`

```
2022/02/17 10:23:58 Cache named cache is created
2022/02/17 10:23:58 Setting key: 443d0058-3035-4b9b-a12f-2a96207c5ff5, value: 4164f81b-718f-4863-8aba-62fa7eea1a4a
2022/02/17 10:23:59 Getting key: 443d0058-3035-4b9b-a12f-2a96207c5ff5
2022/02/17 10:23:59 Lookup resulted in a : HIT
2022/02/17 10:23:59 Looked up value: 4164f81b-718f-4863-8aba-62fa7eea1a4a
2022/02/17 10:23:59 Cache named cache is deleted
```

Closes #71

### CODE REVIEW

**Review by gautamomento — COMMENTED** [review]
Just one comment.

**Comment by gautamomento** [comment]
Can you paste the output produced in the description when running this?

File: golang/go.mod [hunk 1]
```diff
@@ -3,14 +3,17 @@ module github.com/momentohq/go-example
...
+require (
+	github.com/golang-jwt/jwt/v4 v4.3.0 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd // indirect
+	golang.org/x/sys v0.0.0-20220209214540-3681064d5158 // indirect
 	golang.org/x/text v0.3.7 // indirect
-	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 // indirect
-	google.golang.org/grpc v1.43.0 // indirect
+	google.golang.org/genproto v0.0.0-20220217155828-d576998c0009 // indirect
+	google.golang.org/grpc v1.44.0 // indirect
```
[the remark this answers is not in the document]
Comment (rlinehan) [reply golang/go.mod:17]: I think this comes from `go mod` - like we're not managing this ourselves, it happens when you do a `go get -u github.com/momentohq/client-sdk-go` or whatever the syntax actually is and automatically bumps them.

[the remark this answers is not in the document]
Comment (poppoerika) [reply golang/go.mod:17]: This is what I found on the internet about indirect dependencies: https://stackoverflow.com/questions/61115111/how-to-avoid-indirect-dependencies-in-go-mod-file

[the remark this answers is not in the document]
Comment (gautamomento) [reply golang/go.mod:17]: If this isn't hand edited then it seems to be fine.

File: golang/main.go [hunk 2]
```diff
@@ -10,39 +10,37 @@ import (
 )
 
 func main() {
-	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	var authToken = os.Getenv("MOMENTO_AUTH_TOKEN")
```
Comment (gautamomento) [inline golang/main.go:13]: Some of our SDKs do and some don't. But the approach we were planning to take was to add banners to our outputs so the start and end of the example was clear. We have done this in Java and Python. I think while we are at it we should add banners.

Comment (gautamomento) [reply golang/main.go:13]: Actually, I am not sure because we are using log here. So lets skip for now.

**Review by gautamomento — APPROVED** [review]

**Pull request merged**